schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
-CREATE TABLE animals (
-    name varchar(100)
+CREATE TABLE animal (
+    id INT,
```

Comment:
[OPTIONAL] 

- I think it will be better if you can make the` id` column not null and auto-incremented and make it the primary key. This way each row in the table will be uniquely identifiable 👍🏽 

e.g 
    `id int NOT NULL IDENTITY(1,1)`
   `PRIMARY KEY (id)`